### PR TITLE
More updates to make generated app work with Android Studio

### PR DESCRIPTION
**Pull request by bhariharan**
Outstanding Items:

* Tweaking of ```settings.gradle``` (running into issues with inability to use newline character on BSD version of ```sed```).
* Tweaking of generated app's ```build.gradle``` (just because it's a pain in the ass and needs some thought to get foolproof logic around the patterns we search for).
* Copying ```Gradle``` wrapper files over (needs a bit of thought because we don't bundle it in the plugin repo right now).

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
   <dependency id="cordova-plugin-whitelist" />
   <dependency id="cordova-plugin-device" />
-  <dependency id="cordova-plugin-push" url="https://github.com/forcedotcom/phonegap-plugin-push" />
+  <dependency id="phonegap-plugin-push" />
```
Comment (bhariharan) [inline plugin.xml]: I switched to the vanilla plugin for now. If my tests reveal issues, I'll switch to the fork and apply changes there.

**Comment by bhariharan** [comment]
@wmathurin I just realized that you're not a watcher on this repository. That's probably why you haven't seen my previous pull requests. I merged them anyway :D

**Comment by wmathurin** [comment]
Weird that I wasn't a watcher.

**Comment by wmathurin** [comment]
:+1:

**Pull request merged**